gensim/test/test_summarization.py

```diff
@@ -148,6 +148,26 @@ def test_keywords_runs(self):
         kwds_lst = keywords(text, split=True)
         self.assertTrue(len(kwds_lst))
 
+    def test_mz_keywords(self):
+        pre_path = os.path.join(os.path.dirname(__file__), 'test_data')
+
+        with utils.smart_open(os.path.join(pre_path, "head500.noblanks.cor")) as f:
+            text = f.read()
+
+        kwds = mz_keywords(text)
+        self.assertTrue(kwds.startswith('film'))
+        self.assertTrue(kwds.endswith('sought'))
+        self.assertTrue(len(kwds.splitlines()))
+
+        kwds_u = mz_keywords(utils.to_unicode(text))
+        self.assertTrue(len(kwds_u.splitlines()))
+
+        kwds_lst = mz_keywords(text, split=True)
+        self.assertTrue(len(kwds_lst))
+        kwds_auto = mz_keywords(text, scores=True, weighted=False,
+            threshold='auto')
+        self.assertTrue(kwds_auto[-1][1] > 329.0 / 330.0)
```

Comment:
what this "magic" `329.0 / 330.0` means?

Reply:
The document I'm using for testing divides into 329 blocks, when the default block size of 1024 words is used. When threshold='auto', the threshold is calculated as nblocks/(nblocks+1). This is in the docstring for mz_entropy, but I'll add a comment to the test.